crm_lead_stage_monitoring/crm.py

```diff
@@ -35,9 +35,10 @@ def write(self, cr, uid, ids, vals, context=None):
             vals['last_activity_stage'] = fields.datetime.now()
         return super(CrmLead, self).write(cr, uid, ids, vals, context=context)
 
-    def copy(self, cr, uid, id, default=None, context=None):
+    def copy(self, cr, uid, rec_id, default=None, context=None):
```

Comment:
Thanks @yvaucher. I see overwhelming support for the change on that thread so I'll just leave it then. But I do prefer the explicit decorator solution.